package.json

```diff
@@ -19,5 +19,11 @@
 		"redhat.project-initializer",
 		"redhat.vscode-openshift-connector",
 		"vscjava.vscode-java-pack"
+	],
+	"keywords": [
+		"Apache Camel",
```

Comment:
Wouldn't it be best to not have multiple words in a keyword? My suggestion would be to have "Camel", "Fuse" and that's it.

Reply:
It seems that it is adding more ways to find the extension.
I tried with "Apache", currently it returns:
![image](https://user-images.githubusercontent.com/1105127/52219819-b13c3400-289d-11e9-96f9-0b50b61dd882.png)

For Cassandra it seems to be listed because there is "Apache Cassandra" tag: 
![image](https://user-images.githubusercontent.com/1105127/52219848-c2854080-289d-11e9-88af-614895ac1185.png)

in case we reach the 5 keyword limit, I think that it will the first to remove but currently it seems to give a bit more of visibility.

Reply:
So why not just add a Apache tag then?

Reply:
if you think that it is still not a good idea. I can use the single-words in this PR and then create an issue to discuss about other potential keywords.

Reply:
I would try to only use single words instead of groups of words. So I am fine with adding "Apache" to the list of keywords.

Reply:
> So why not just add a Apache tag then? (additionally)

It won't change the search results, as when user will type "Apache", the extension will be found due to the "Apache Camel" keyword.

Reply:
Go for whatever u think is best. We can tweak the tags later on if required.